### PR TITLE
READY : Covering option `dry`.

### PR DESCRIPTION
**Pull request by JackieJoo**
1. Extended test routine `startOptionDry`.
    startOptionDry, /* qqq for Yevhen : make sure option dry is covered good enough. */
    /* qqq for Yevhen : write test routine startOptionDryMultiple */
2. Wrote test routine `startSingleOptionDry`.
3. Wrore test routine `startOptionDryMultiple`.
4. Added check for option `dry` in routine `run2`.
5. Fixed according to the review.
6. Extended test coverage.
7. Bug in routine `run2`, left a comment.
if( o.sync && !o.deasync )
      {
        /* When dry : 1, o.process = null */
        if( o.process.error )
        handleError( o.process.error );
        else
        handleClose( o.process.status, o.process.signal );
      }

### CODE REVIEW

File: proto/wtools/abase/l4.test/ProcessBasic.test.s [hunk 1]
```diff
@@ -21597,60 +21597,87 @@ function startOptionStreamSizeLimitThrowing( test )
 }
 
 //
-
 function startOptionDry( test )
 {
   let context = this;
   let a = context.assetFor( test, false );
-  let programPath = a.program( testApp );
+  let programPath = a.path.nativize( a.program( testApp ) );
```
Comment (Wandalen) [inline proto/wtools/abase/l4.test/ProcessBasic.test.s:21604]: redundant

File: proto/wtools/abase/l4_process/l3/Execution.s [hunk 2]
```diff
@@ -1735,6 +1735,7 @@ function start_body( o )
       readies.push( o2.ready );
 
       if( o.streamOut || o.streamErr )
+      if( !o.dry )
```
Comment (Wandalen) [inline proto/wtools/abase/l4_process/l3/Execution.s:1738]: put before prev if

File: proto/wtools/abase/l4.test/ProcessBasic.test.s (hunk 1 above)
Comment (JackieJoo) [reply proto/wtools/abase/l4.test/ProcessBasic.test.s]: Removed

File: proto/wtools/abase/l4_process/l3/Execution.s (hunk 2 above)
Comment (JackieJoo) [reply proto/wtools/abase/l4_process/l3/Execution.s:1737]: Done

**Comment by Wandalen** [comment]
No test routines duplication.

**Comment by Wandalen** [comment]
Each usage of option o.dry should be referenced in tests.

**Comment by JackieJoo** [comment]
> Each usage of option o.dry should be referenced in tests.

Done

**Comment by Wandalen** [comment]
I didn't mean that. Tests does not cover all branches where o.dry has influence.

**Pull request merged**